src/components/Order/index.tsx

```diff
@@ -139,7 +142,12 @@ const defaultOrderTypes: DropdownElem[] = [
 const splitBorder = 449;
 const defaultWidth = 635;
 
-class Order extends React.PureComponent<OrderComponentProps> {
+
+class Order extends React.PureComponent<OrderComponentProps, State> {
+    public state = {
+        index: 0,
+    };
```

Comment:
tabIndex is the Prop of component/Orders soo you need to define index in containers/Orders state